docs/walkthrough/index.md

```diff
@@ -0,0 +1,328 @@
+
+## Install Grafana
+
+Grafana's chart has a few default values you may not want long term - persistant storage is disabled and admin username/password is randomly generated - but for our sample the out-of-the-box install will work. 
+
+Run `helm install stable/grafana --name grafana` and you should see output that includes this command:
+
+```
+kubectl get secret --namespace default grafana -o jsonpath="{.data.admin-password}" | base64 --decode ; echo
+```
+
+Run this to get your Grafana password.
+
+Now you can use `kubectl port-forward` again to log in to your Grafana dashboard. `kubectl port-forward svc/grafana 8080:80` will make your dashboard available at http://localhost:8080, and you can log in with username 'admin' and the password you retrieved.
+
+## Add Prometheus as a data source
+
+After logging in, you should see an option to "Add a Data Source." Click that, and choose the Prometheus source type (if it's not immediately visible, search for it).
+
+The only setting you should need to edit here is the URL, under the HTTP section. Within your cluster, `http://<prometheus-release-name>-prometheus-server.default.svc.cluster.local` should resolve to the Prometheus server service. (Default in that URL refers to the namespace - if you installed in a namespace other than default, change that.)
+
+Set your service's name as the Prometheus URL in Grafana, and save the data source. It should tell you that the data source is working.
+
+## Create a Grafana dashboard for queue metrics
+
+First, we'll make a basic dashboard with the metric we set up in Promitor. Then you can use a pre-built dashboard to show Prometheus' default Kubernetes metrics.
+
+Go to the + button on the sidebar, and choose "Dashboard". To make a simple graph showing your queue size, you can write `demo_queue_size` in the query field. Click out of that input field and the graph should update.
+
+To see more, you can go back to Service Bus Explorer and send or receive messages. Your Grafana graph won't update immediately, but you should see results in a few minutes. 
+
+In order to see results without manually refreshing, find the dropdown menu in the top right corner that sets the time range of the graph. Here you can edit time range and refresh rate.
+
+Make sure to save your new dashboard before exiting the page.
+
+## Import an existing Grafana dashboard
+
+Now we'll import a pre-created dashboard that shows Kubernetes metrics. There are multiple available on Grafana Lab's dashboard site - try [6417](https://grafana.com/dashboards/6417).
+
+To import a dashboard, click the + button on the sidebar and choose "Import." From there, you can either load a JSON file or enter the dashbord ID: 6417.
+
+Click "Load" and you will be given some configuration options. The only one that needs to be set is the Prometheus data source. From there, you should be able to create the dashboard and view metrics about your AKS cluster.
+
+## Set up a Prometheus alert
```

Comment:
See my other suggestion to create a separate issue for this to relieve you. Thoughts?

Reply:
That would actually be awesome, I have a lot on my plate at the moment. Once I update the grafana section I'll remove the WIP tag & make an issue for alerting.

Reply:
Sounds great, thanks! And take some time off ;)